extensions/amp-render/1.0/amp-render.js

```diff
@@ -185,6 +185,14 @@ export class AmpRender extends BaseElement {
     });
   }
 
+  /** @override */
+  buildCallback() {
+    super.buildCallback();
+    if (!this.element.hasAttribute('aria-live')) {
+      this.element.setAttribute('aria-live', 'polite');
+    }
+  }
```

Comment:
Must have missed this in this review ... Bento components should not have `buildCallback`, please move to `init`.